src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -470,7 +470,7 @@ def get(self):
 @ControllerDoc("Dashboard UI helper function; not part of the public API", "CephFSUi")
 class CephFsUi(CephFS):
     RESOURCE_ID = 'fs_id'
-
+    
```

Comment:
This can be removed.

Reply:
Resolved; line removed and white space errors fixed